controllers/new.go

```diff
@@ -300,6 +303,75 @@ func (k *kubeResources) oldPolicyEngineRoute(instance *cv1.UpdateService) *route
 	}
 }
 
+func (k *kubeResources) newNetworkPolicy(instance *cv1.UpdateService) *networkingv1.NetworkPolicy {
+	return &networkingv1.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      instance.Name,
+			Namespace: instance.Namespace,
+			Annotations: map[string]string{
+				DescriptionAnnotation: "This NetworkPolicy allows all egress, to support graph-builder scraping and DNS. " +
+					"It allows ingress from the router, to support serving policy-engine responses. " +
+					"All other ingress is blocked, including, for now, metrics scraping.",
+			},
+			Labels: map[string]string{
+				"app": instance.Name,
+			},
+		},
+		Spec: networkingv1.NetworkPolicySpec{
+			PodSelector: metav1.LabelSelector{
+				MatchLabels: map[string]string{
```

Comment:
```suggestion
                // The labels defined here match ones on the OSUS's pod
				MatchLabels: map[string]string{
```

Reply:
Catching public GitHub up with some internal discussion, I think the:

```
PodSelector: metav1.LabelSelector{
  MatchLabels: map[string]string{
    "app": nameDeployment(instance),
  },
},
```

is already saying "The labels defined here match ones on the OSUS's Pod" pretty clearly.  But I'm not opposed to comments or other refactoring to make that connection clearer, if other folks want to open pulls with that.